docs/start/smesher/trusted_state_quicksync.md

```diff
@@ -0,0 +1,45 @@
+---
+id: trusted_state_quicksync
+title: Trusted State aka Quicksync
+---
+
+## Quicksync
+
+In response to the growing difficulty of syncing a fresh node, we have prepared a way to speed up the syncing process. Introducing, Quicksync.
+
+Before reading any further, plaese read the annoucement about Quicksync [here](https://spacemesh.io/blog/making-sync-faster/).
+
+### Getting the archive automatically
+
+#### Smapp
+
+Smapp will propose to run Quicksync automatically.
+
+#### CLI
+
+We have prepared a simple tool called [quicksync-rs](https://github.com/spacemeshos/quicksync-rs) that simplifies the process of getting the archive.
```

Comment:
The quicksync README lacks information:
- how to install it
- how to run it (i.e. a command to copy-paste and execute)

I think the readme should be extended first.